sg.tf

```
@@ -0,0 +1,24 @@
+resource "aws_security_group" "allow_acesso" {
```

```suggestion
resource "aws_security_group" "allow_access" {
```

Mais uma padronização.